examples/animation-generic-logo/index.html

```diff
@@ -19,7 +19,7 @@
           <a-animation attribute="scale" from="0 0 0" to="1 1 1" begin="500" dur="500" fill="both" easing="ease-out"></a-animation>
           <a-animation attribute="position" from="0 -0.5 0" to="0 0 0" begin="500" dur="500" fill="both" easing="ease-out"></a-animation>
           <a-animation attribute="rotation" from="0 -300 0" to="30 54 36" begin="500" dur="1200" fill="both" easing="ease-out"></a-animation>
-          <a-cube width="0.25" depth="0.25" height="0.25" color="#EF2D5E"></a-cube>
+          <a-box width="0.25" depth="0.25" height="0.25" color="#EF2D5E"></a-box>
           <a-frame scale="0.04 0.04 0.04"></a-frame>
```

Comment:
<img width="1363" alt="screenshot 2016-02-18 11 48 56" src="https://cloud.githubusercontent.com/assets/203725/13156080/962351ac-d635-11e5-8d91-c7316039fa88.png">
<img width="1363" alt="screenshot 2016-02-18 11 48 43" src="https://cloud.githubusercontent.com/assets/203725/13156079/96233762-d635-11e5-9a5f-954ca33f47ac.png">



since the `<a-frame>` template definition is removed, feel free to replace with this:

```html
<!-- an actual frame … yes, an a-frame ;-) -->
<a-entity scale="0.04 0.04 0.04">
  <a-box position="5 5 0" rotation="0 0 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="5 -5 0" rotation="0 0 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="-5 5 0" rotation="0 0 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="-5 -5 0" rotation="0 0 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>

  <a-box position="0 5 5" rotation="0 90 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="0 5 -5" rotation="0 90 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="0 -5 5" rotation="0 90 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="0 -5 -5" rotation="0 90 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>

  <a-box position="5 0 5" rotation="90 0 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="5 0 -5" rotation="90 0 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="-5 0 5" rotation="90 0 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
  <a-box position="-5 0 -5" rotation="90 0 0" width="0.1" height="0.1" depth="10" color="#404040"></a-box>
</a-entity>
```